mitxpro/settings.py

```diff
@@ -57,6 +57,20 @@
     description="Application-level SSL redirect setting.",
 )
 
+ZENDESK_CONFIG = {
+    "HELP_WIDGET_ENABLED": get_string(
+        "ZENDESK_HELP_WIDGET_ENABLED",
+        False,
```

Comment:
`ZENDESK_HELP_WIDGET_ENABLED` will be used for enabling and disabling zendesk help web widget now.